work-locally.mdx

```diff
@@ -3,28 +3,35 @@ title: Work Locally
 description: "Seamless local-to-cloud experience for rapid experimentation"
 ---
 
-Building Modus apps is even easier when you add Hypermode. The Hyp CLI extends
-the [local Modus experience](/modus/modus-cli) with access to Hypermode-hosted
-models defined in your [app's manifest](/modus/app-manifest) for rapid
+By using the Hyp CLI, you can access Hypermode-hosted models defined in your
+[app's manifest](/modus/app-manifest). This allows for seamless and rapid
 experimentation with new AI models.
 
 ## Hyp CLI setup
 
-For access to hosted models, install the Hyp CLI to augment Modus.
+The Hyp CLI helps you access Hypermode-hosted models and manage your Hypermode
+projects. You can install the Hyp CLI to augment Modus and access your
+Hypermode-hosted models locally.
```

Comment:
I would be clearer that your models aren't running locally:

"and invoke your Hypermode-hosted models from your local Modus development environment"